modules/openapi-generator/src/main/resources/JavaJaxRS/README.mustache

```diff
@@ -1,22 +1,23 @@
-# Swagger Jersey generated server
+# JAX-RS/Jersey server with OpenAPI
 
 ## Overview
-This server was generated by the [OpenAPI Generator](https://openapi-generator.tech) project. By using the
-[OpenAPI-Spec](https://openapis.org) from a remote server, you can easily generate a server stub.  This
-is an example of building a OpenAPI-enabled JAX-RS server.
+This server was generated by the [OpenAPI Generator](https://openapi-generator.tech) project. By using an
+[OpenAPI-Spec](https://openapis.org), you can easily generate a server stub.
 
+This is an example of building a OpenAPI-enabled JAX-RS server.
 This example uses the [JAX-RS](https://jax-rs-spec.java.net/) framework.
+Jersey is used as JAX-RS implementation, `io.swagger:swagger-jersey2-jaxrs` is used to derive the OpenAPI Specification from the annotated code.
 
 To run the server, please execute the following:
 
 ```
 mvn clean package jetty:run
 ```
 
-You can then view the swagger listing here:
+You can then view the OpenAPI v2 specification here:
 
 ```
-http://localhost:{{serverPort}}{{contextPath}}/openapi.json
+http://localhost:{{serverPort}}{{contextPath}}/swagger.json
```

Comment:
@jmini Shall we rename 'swagger.json' to 'openapi.json` similar to what we did in other server generators?

Reply:
As far as I know, this is not something that can be defined, the server is not serving a static file but is deriving the spec from the annotation in the code. So unless #27 is solved, we serve a OASv2 under `/swagger.json`.

Reply:
Ok. Thanks for the explanation